buildSrc/src/main/groovy/com/uber/okbuck/core/model/AndroidTarget.groovy

```diff
@@ -257,7 +261,8 @@ abstract class AndroidTarget extends JavaLibTarget {
             XmlSlurper slurper = new XmlSlurper()
             GPathResult manifestXml = slurper.parse(project.file(mergedManifest))
 
-            manifestXml.@package = applicationId + applicationIdSuffix
+            packageName = manifestXml.@package
```

Comment:
If you look at the intermediate manifest gradle generates, it has applicationId + suffix in it

Like
```
<manifest xmlns:android="http://schemas.android.com/apk/res/android"
    package="com.uber.okbuck.example.demo.debug"
```